### ISSUE TITLE
Upgrade seedrandom

### ISSUE THREAD

**Issue by meandavejustice** [mozilla/testpilot#3862]
Getting a csp error that `eval` is being used with new version.

**Comment by meandavejustice** [comment]
seedrandom now uses `eval` in order to 
> Detect the global object, even if operating in strict mode.

https://github.com/davidbau/seedrandom/blob/released/seedrandom.js#L32

The current version is working fine for us, we can just leave it as is or fine a replacement eventually.

**Comment by lmorchard** [comment]
I want to say seedrandom is related to variant test support we used to have. I wonder if it could go away?

**Comment by meandavejustice** [comment]
Yeah, only place we are using it is in a reducer for varianttests. https://github.com/mozilla/testpilot/blob/f8a4e4521574c5ee90fcee6c3d96a5fd5f7f725d/frontend/src/app/reducers/varianttests.js#L39
